CITATION.cff

```diff
@@ -14,14 +14,19 @@ authors:
     family-names: Liu
     orcid: 'https://orcid.org/0000-0002-1966-8460'
     affiliation: Netherlands eScience Center
+- given-names: Keith
```

Comment:
Yeah, this is correct! Thanks for adding me!

(Looks like this line lost its indent.)

```suggestion
   - given-names: Keith
```

Reply:
well spotted :)